databases/mycli/Portfile

```diff
@@ -1,14 +1,12 @@
-# $Id$
-
 PortSystem          1.0
 PortGroup           python 1.0
 PortGroup           github 1.0
 
-github.setup        dbcli mycli 1.5.2 v
+github.setup        dbcli mycli 1.8.1 v
 
-python.default_version     27
+python.versions     27 34 35
```

Comment:
> what should I do with `python.versions` and `python.default_version`?

Either nothing (that is: remove the patch for this line) or perhaps something similar like some other ports do. A random example:
~~~tcl
variant python26 conflicts python27 description {Use Python 2.6} {}
variant python27 conflicts python26 description {Use Python 2.7} {}
if {[variant_isset python26]} {
    python.default_version 26
} else {
    default_variants +python27
    python.default_version 27
}
~~~
Of course you should do this for the suitable versions of python and only if you care about supporting different versions.